js/main.js

```diff
@@ -152,14 +158,11 @@ function updateFeedback(response, player) {
       <span>${subchunk.toUpperCase()}! ${playerName} is back in the game!</span>
     `
     undoSingleDeal()
-    toggleHighlighting(player)
   } else if (response === 'winner') {
     chunk = `
       <span>${response.toUpperCase()}! ${playerName} wins the game!</span>
     `
 
-    checkLocalStorage();
-
     player1front.classList.remove('hidden');
     player1empty.classList.add('hidden');
```

Comment:
instead of the chunks, create a function, pass in the phrase, and use the function to display it. it won't change much, but may lose a couple of lines of code within this function